src/Gui/ToolBarManager.cpp

```diff
@@ -182,6 +182,12 @@ void ToolBarManager::setup(ToolBarItem* toolBarItems)
     static QPointer<QWidget> _ActionWidget;
     if (!_ActionWidget) {
         _ActionWidget = new QWidget(getMainWindow());
+        _ActionWidget->setObjectName(QStringLiteral("_fc_action_widget_"));
+		/* Although _ActionWidget has zero size, it somehow has a

```

Comment:
@Syres916 can you kindly add `TODO` line here explaining this is temporary hack till we figure out a broader solution ?